.travis.yml

```diff
@@ -6,5 +6,8 @@ python:
 install:
   - "pip install pipenv"
   - "pipenv install --dev --skip-lock"
+  - "pipenv install coveralls"
```

Comment:
IMO this coveralls stuff should be a totally separate PR but...

You shouldn't use `pipenv` install this dependency here.
1. Either add it to the `Pipfile` so that it gets installed along with the `dev` dependencies
2. install it directly with pip in the first install step. `pip install pipenv coveralls`